pyogrio/geopandas.py

```diff
@@ -127,10 +128,12 @@ def read_dataframe(
     fid_as_index : bool, optional (default: False)
         If True, will use the FIDs of the features that were read as the
         index of the GeoDataFrame.  May start at 0 or 1 depending on the driver.
-    use_arrow : bool, default False
+    use_arrow : bool, optional (default: False)
         Whether to use Arrow as the transfer mechanism of the read data
         from GDAL to Python (requires GDAL >= 3.6 and `pyarrow` to be
         installed). When enabled, this provides a further speed-up.
+    arrow_to_pandas_kwargs: dict, optional (default: None)
+        When use_arrow is True, these kwargs will be passed to the _TO_PANDAS call.
```

Comment:
```suggestion
        When use_arrow is True, these kwargs will be passed to the ``to_table()`` call for the arrow to pandas conversion .
```

Reply:
The method is called [to_pandas()](https://arrow.apache.org/docs/python/generated/pyarrow.Table.html#pyarrow.Table.to_pandas). I gave it a quick try, but it seems that a link cannot be visualized like this ``      ``.

Reply:
Whoops, sorry, that was meant to be `to_pandas()` (I mostly wanted to comment that we should not use `_TO_PANDAS`)